fudge-ratio/src/main/webapp/calculator/index.html

```diff
@@ -12,7 +12,7 @@
   <body>
     <h1>Fudge Ratio Calculator</h1>
     <p>
-      Select a estimated time, a task and click the start button when you're
+      Select an estimated time and a task, then click the start button when you're
```

Comment:
Thanks so much for this, #workingOnWriting

Reply:
I was just being nit-picky!